src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -177,7 +177,7 @@ public void Start()
                     // Make sure the HttpListener instance can't be used if Start() failed.
                     _state = State.Disposed;
                     DisposeInternal();
-                    Logger.LogError(0, exception, "Start");
+                    Logger.LogError(LoggerEventIds.ListenerStartException, exception, "Start");
```

Comment:
Be consistent about Error vs Exception vs Failed/Failure. Let's use Error as it's simpler.